cfme/common/provider_views.py

```diff
@@ -249,15 +251,15 @@ class ProviderTemplatesView(ProviderVmsTemplatesView):
 
     @property
     def is_displayed(self):
-        if self.browser.product_version < '5.10':
-            title = '{name} (All VM Templates)'.format(name=self.context['object'].name)
-        else:
-            title = '{name} (All VM Templates and Images)'.format(name=self.context['object'].name)
-
+        title = VersionPicker({
+            Version.lowest(): '{name} (All VM Templates)'.format(name=self.context['object'].name),
+            '5.10': '{name} (All VM Templates and Images)'.format(name=self.context['object'].name),
+        }).pick(self.extra.appliance.version)
```

Comment:
**Question** Does this still work without calling `.pick` ? I am thinking it may not be needed here

Reply:
@jawatts I tried and it didn't. After that I added `.pick`

Reply:
@lina-nikiforova Thanks, I was curious!